src/theme/book.js

```diff
@@ -340,6 +340,23 @@ function playpen_text(playpen) {
         }
     });
 
+    // Toggle sidebar
```

Comment:
Need to move the non-RFC changes to a different PR

Reply:
Ah, my bad, I've branched it off the restyled branch.